### PR TITLE
Docs/diataxis

### PR DESCRIPTION
**Pull request by aleksei-korolev**
# Description of Change

Reorganize to the new structure.

## Links to Any Relevant Issues

I would expect to see the "Verify Firefly Download" anywhere but this space tbh. Unfortunately, this is also a very "hot" article (due to the reference on firefly.iota.org I believe). Before we merge this PR we should at the very least setup a redirect. Ideally, we find this guide a better place and also update the link on the firefly.iota.org afterwards.

## Type of Change

Choose a type of change, and delete any options that are not relevant.

- Documentation Enhancement

## How the Change Has Been Tested

I ran it locally

## Change Checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [X] I have followed the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have tested the docs locally

### CODE REVIEW

File: docs/how_tos/migration/token_migration.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Firefly will initially only be available on desktop operating systems such as: m
 
 For further information on the migration process, see our [blog post](https://blog.iota.org/firefly-token-migration/).
 
-For a detailed explanation on how the migration process works technically, see the [migration-mechanism](./migration_mechanism) page.
```
Comment (lucas-tortora) [inline docs/how_tos/migration/token_migration.md:46]: this doc is tagged as expanation

File: docs/how_tos/migration/hub_migration.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ IOTA Hub was deprecated with the Chrysalis upgrade and will not work with the ne
 
 To upgrade from Hub to a Chrysalis implementation you will need to:
 
- - Integrate the Chrysalis network using [Wallet.rs](https://wiki.iota.org/wallet.rs/welcome) as mentioned in the [Exchange section](exchange.md).
+ - Integrate the Chrysalis network using [Wallet.rs](https://wiki.iota.org/wallet.rs/welcome) as mentioned in the [Exchange section](../exchange.md).
```
Comment (lucas-tortora) [inline docs/how_tos/migration/hub_migration.md:37]: this docs is tagged as explanation

File: docs/explanations/update/chrysalis_improvements.md [hunk 3]
```diff
@@ -1,59 +1,39 @@
----
-keywords:
-- Coordicide
-- Upgrades
-- Mainnet
-- Protocol Improvements
-- explanation
-description: Specific Chrysalis Upgrades from IOTA 1.0.  
-image: /img/logo/Chrysalis_logo_dark.png
----
```
Comment (lucas-tortora) [inline docs/explanations/update/chrysalis_improvements.md:1]: meta tags are missing here.

**Review by lucas-tortora — APPROVED** [review]
Just a few missing tags :) LGTM

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**